extras/tools/bin2ota.py

```diff
@@ -26,6 +26,8 @@
     magic_number = 0x2341025B.to_bytes(4,byteorder='little')
 elif board == "NANO_RP2040_CONNECT":
     magic_number = 0x2341005E.to_bytes(4,byteorder='little')
+elif board == "ESP32":
+    magic_number = 0x45535033.to_bytes(4,byteorder='little')
```

Comment:
Is there the same VID/PID for all ESP32 boards? Somehow this is hard to believe, unless I'm missing something here.

Reply:
You are correct, there is really a huge amount of VID/PID pairs in case of ESP boards. For this reason this is intended as a generic magic number not a VID/PID combination. I'll add a note to make it clear.

The Updater class however will check for the binary compatibility using the ESP binary header

https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/system/app_image_format.html#structures

and checking if the `chip_id` , that differs from ESP32 to ESP32-S2 and ..., is matching.

Reply:
Roger that, thank you for the clarification ;)